django/db/backends/base/features.py

```diff
@@ -298,6 +298,9 @@ class BaseDatabaseFeatures:
     # during INSERT?
     supports_update_conflicts = False
     supports_update_conflicts_with_target = False
+    # False by default; override in backends that support conditional WHERE
+    # in bulk updates.
+    supports_update_conflicts_where = False
```

Comment:
And then this to `supports_update_conflicts_with_condition`... 🤔 

This all sounds better to me - sorry, I know it's another rename, but sometimes the passage of time helps! 😅

Reply:
I renamed  😅